genMarketplace.js

```diff
@@ -46,6 +50,61 @@ function capitalizeFirstLetter(string) {
   }
 }
 
+function checkURLAndModifyLink(url, listItem) {
+  // Check if a README file exists in the given URL. If so, attach it to the relevant content item.
+  return fetch(url).then((response) => {
+    if (response.ok) {
+      listItem.docLink = url;
+    }
+    else {
+      listItem.docLink = ""
+    }
+  }).catch(err => listItem.docLink = "");
+}
+
+function normalizeItemName(itemName) {
```

Comment:
note this does not work properly
for example, the link from 
![image](https://user-images.githubusercontent.com/31018228/123758566-b8799780-d8c7-11eb-90ce-2a4302c29cf9.png)
leads to https://xsoar.pan.dev/docs/reference/integrations/mitre-attck-feed which does not exist
in general im not fan of having this normalization functionality in both python and javascript

Reply:
@Itay4 I think you were looking at the old preview site, in the new one there aren't supposed to be broken links 
![image](https://user-images.githubusercontent.com/50294648/123769055-9127c800-d8d1-11eb-9bb0-eef96c52a297.png)
 But I do see there is a README for this which we don't link to.
I tried using the same library the python's logic but it resulted differently.
I think I will create a file containing all the links (created be the python logic) and will maintain it.